src/components/App.js

```diff
@@ -0,0 +1,14 @@
+import '../App.css';
+import Display from './Display';
+import ButtonPannel from './ButtonPannel';
+
+function App() {
+  return (
```

Comment:
- [ ] Also kindly use ES6 arrow function syntax. Do this for other affected modules too.